base/applications/fltmc/lang/es-ES.rc

```diff
@@ -2,30 +2,30 @@ LANGUAGE LANG_SPANISH, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
-    IDS_USAGE "\nValid commands :\n\
-               load        Loads a Filter driver\n\
-               unload      Unloads a Filter driver\n\
-               filters     Lists the Filters currently registered in the system\n\n\
-               volumes     Lists all volumes\n\n"
+    IDS_USAGE "\nComandos válidos :\n\
+               load        Carga un controlador de filtro\n\
+               unload      Des-carga un controlador de filtro\n\
+               filters     Listar los Filtros que estén actualmente registrados en el sistema\n\n\
+               volumes     Listar todos los volúmenes\n\n"
 
-    IDS_USAGE_LOAD      "Loads a filter driver\n"
-    IDS_USAGE_UNLOAD    "Unloads a filter driver\n"
-    IDS_USAGE_FILTERS   "Lists all registered filters\n"
-    IDS_USAGE_VOLUMES   "Lists all volumes\n"
+    IDS_USAGE_LOAD      "Carga un controlador de filtro\n"
+    IDS_USAGE_UNLOAD    "Des-carga a controlador de filtro\n"
+    IDS_USAGE_FILTERS   "Listar los Filtros registrados\n"
+    IDS_USAGE_VOLUMES   "Listar todos los volúmenes\n"
 END
 
 STRINGTABLE
 BEGIN
-    IDS_DISPLAY_FILTERS1    "Filter Name                     Num Instances    Altitude    Frame\n"
-    IDS_DISPLAY_FILTERS2    "Filter Name                     Num Instances   Frame\n"
-    IDS_DISPLAY_VOLUMES     "Dos Name                        Volume Name                              FileSystem   Status\n"
+    IDS_DISPLAY_FILTERS1    "Nombre Filtrado                     Num Instancias    Altitud    Frame\n"
```

Comment:
Please double-check, when testing the program, that these column names are correctly being displayed aligned on screen.

Reply:
Done, I have fixed the lines and now it's aligned.